files/build_templates/sonic_debian_extension.j2

```diff
@@ -731,6 +731,27 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT docker $SONIC_NATIV
 echo "docker images pull complete"
 {% endif %}
 
+{% if include_kubernetes_master == "y" %}
+# Pull in kubernetes master docker images
+echo "pulling kubernetes master images ..."
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT docker $SONIC_NATIVE_DOCKERD_FOR_DOCKERFS pull k8s.gcr.io/pause:${MASTER_PAUSE_VERSION}
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT docker $SONIC_NATIVE_DOCKERD_FOR_DOCKERFS pull k8s.gcr.io/kube-apiserver:v${KUBERNETES_VERSION}
```

Comment:
>KUBERNETES_VERSION [](http://example.com/codeflow?start=137&length=18)

Is it MASTER_KUBERNETES_VERSION?

Reply:
Yes, it should be MASTER_KUBERNETES_VERSION, I copied the wrong line. Have corrected it to MASTER_KUBERNETES_VERSION.